src/nodeattr/nodeattr.c

```diff
@@ -76,6 +76,8 @@ static struct option longopts[] = {
     { "diff", 1, 0, 'd'},
     { "expand", 0, 0, 'e'},
     { "compress", 0, 0, 'C'},
+    { "compress-attrs", 0, 0, 'C'},
+    { "compress-hosts", 0, 0, 'H'},

```

Comment:
not necessary, just there for convenience as a parallel to --compress-hosts.  I'll be sure to mention it as an alias to --compress in documentation.